src/Raven.Server/Documents/Handlers/Admin/AdminLogsHandler.cs

```diff
@@ -77,5 +84,68 @@ public async Task RegisterForLogs()
                 await LoggingSource.Instance.Register(socket, context, ServerStore.ServerShutdown);
             }
         }
+
+        [RavenAction("/admin/logs/download", "GET", AuthorizationStatus.Operator)]
+        public async Task Download()
+        {
+            var contentDisposition = $"attachment; filename={DateTime.UtcNow:yyyy-MM-dd H:mm:ss} - Node [{ServerStore.NodeTag}] - Logs.zip";
+            HttpContext.Response.Headers["Content-Disposition"] = contentDisposition;
+            HttpContext.Response.Headers["Content-Type"] = "application/zip";
+
+            var adminLogsFileName = $"admin.logs.download.{Guid.NewGuid():N}";
+            var adminLogsFilePath = ServerStore._env.Options.DataPager.Options.TempPath.Combine(adminLogsFileName);
+
+            var from = GetDateTimeQueryString("from", required: false);
+            var to = GetDateTimeQueryString("to", required: false);
+
+            using (var stream = new TempFileStream(SafeFileStream.Create(adminLogsFilePath.FullPath, FileMode.CreateNew, FileAccess.ReadWrite, FileShare.ReadWrite, 4096,
+                       FileOptions.DeleteOnClose | FileOptions.SequentialScan)))
```

Comment:
DeleteOnClose does not work properly on Linux. Create 'DeleteOnCloseFileStream' maybe (we have DeleteOnCloseZipArchive already) and handle it properly.